core/Archive/ArchiveInvalidator.php

```diff
@@ -544,12 +544,15 @@ public function removeInvalidations($idSite, $plugin, $report = null)
      * since adding invalidations can take a long time and delay UI response times.
      *
      * @param int|int[]|'all' $idSites
-     * @param string $pluginName
+     * @param string|int $pluginName
```

Comment:
allowed int since mysqli might return `int` in some cases. We can convert to `string` though if needed eg in Funnels etc